examples/seeder_example/vt_application.hpp

```diff
@@ -0,0 +1,255 @@
+	static const std::array<ActiveScreen, static_cast<std::uint8_t>(Alarm::NumberOfAlarms)> ALARM_SCREENS; ///< Used to convert index of alarm to active screen
+	static const std::array<std::uint16_t, static_cast<std::uint8_t>(Alarm::NumberOfAlarms)> ALARM_DESCRIPTION_LINES; ///< Stores object IDs corresponding to each alarm's one line description
+
+	SectionControlImplementSimulator sectionControl; ///< A class that manages section control
+	isobus::ProcessingFlags txFlags; ///< A set of flags to handle transmitting VT messages and retries as needed
+	std::vector<std::uint8_t> objectPool; ///< Stores our object pool
+	std::array<AlarmMaskCondition, static_cast<std::uint8_t>(Alarm::NumberOfAlarms)> alarmConditions; ///< Tracks alarm conditions in priority order
+	std::shared_ptr<void> softkeyEventListener = nullptr; ///< Stores a handle for soft key event callbacks
+	std::shared_ptr<void> buttonEventListener = nullptr; ///< Stores a handle for button event callbacks
+	std::shared_ptr<void> numericValueEventListener = nullptr; ///< Stores a handle for numeric value event callbacks
+	float currentBusload = 0.0f; ///< Stores the current CAN bus load, used to detect changes and update the UT value
+	ActiveScreen currentlyActiveScreen = ActiveScreen::Main; ///< The currently active data mask
+	ActiveScreen previousActiveScreen = ActiveScreen::Main; ///< Stores the previous active screen so it can be returned to if needed
+	Statistics currentlySelectedStatistic = Statistics::None; ///< The currently selected statistic on the stats data mask
+	SpeedSources currentSpeedSource = SpeedSources::MachineSelected; ///< Keeps track of which speed source is currently being used
+	isobus::LanguageCommandInterface::DistanceUnits distanceUnits = isobus::LanguageCommandInterface::DistanceUnits::Metric; ///< Stores the current displayed distance units
+	isobus::VirtualTerminalClient::VTVersion utVersion = isobus::VirtualTerminalClient::VTVersion::ReservedOrUnknown; ///< Stores the UT's version for display in the pool
+	isobus::TaskControllerClient::Version tcVersion = isobus::TaskControllerClient::Version::Unknown; ///< Stores the TC's version for display in the object pool
+	isobus::SpeedMessagesInterface speedMessages; ///< Interface for reading speed from the bus
+	std::shared_ptr<isobus::DeviceDescriptorObjectPool> ddop = nullptr; ///< Stores our application's DDOP
+	std::shared_ptr<std::function<void(const std::shared_ptr<isobus::SpeedMessagesInterface::MachineSelectedSpeedData> &, const bool &)>> machineSelectedSpeedEventHandle; ///< Handle for MSS events
+	std::shared_ptr<std::function<void(const std::shared_ptr<isobus::SpeedMessagesInterface::GroundBasedSpeedData> &, const bool &)>> groundBasedSpeedEventHandle; ///< Handle for ground based speed events
+	std::shared_ptr<std::function<void(const std::shared_ptr<isobus::SpeedMessagesInterface::WheelBasedMachineSpeedData> &, const bool &)>> wheelBasedSpeedEventHandle; ///< Handle for wheel based speed events
```

Comment:
Maybe use `std::shared_ptr<void>` here for nicer formatting on smaller screen, as it doesn't really matter what the function definition is here